spark-bigquery-dsv2/spark-3.5-bigquery/pom.xml

```diff
@@ -0,0 +1,51 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.cloud.spark</groupId>
+    <artifactId>spark-bigquery-dsv2-parent</artifactId>
+    <version>${revision}</version>
+    <relativePath>../spark-bigquery-dsv2-parent</relativePath>
+  </parent>
+
+  <artifactId>spark-3.5-bigquery</artifactId>
+  <version>${revision}</version>
+  <name>BigQuery DataSource v2 for Spark 3.5</name>
+  <properties>
+    <spark.version>3.5.0</spark.version>
+    <shade.skip>false</shade.skip>
+  </properties>
+  <licenses>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+  <dependencies>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>spark-3.5-bigquery-lib</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-avro_2.12</artifactId>
```

Comment:
should be scala 2.13

Reply:
It doesn't compile if I use 2.13. spark-3.4-bigquery also uses 2.12 here.

Reply:
Fails here:
```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /Users/vkarve/Documents/oss-repos/spark-bigquery-connector/spark-bigquery-dsv2/spark-3.5-bigquery/src/test/java/com/google/cloud/spark/bigquery/integration/Spark35IndirectWriteIntegrationTest.java:[31,10] cannot access scala.Serializable
  class file for scala.Serializable not found
```